internal/database/sqlite.go

```diff
@@ -113,6 +113,27 @@ PRAGMA foreign_keys=on;`,
 			"add_holder_name_to_accounts",
 			`alter table accounts add column holder_name default '';`,
 		),
+		execsql(
+			"create_validations",
+			`create table if not exists validations(
+				validation_id primary key,
+				account_id,
+				status,
+				strategy,
+				vendor,
+				created_at datetime,
+				updated_at datetime
+			);`,
+		),
+		execsql(
+			"create validations index",
+			`
+			create unique index
```

Comment:
From https://github.com/moov-io/customers/pull/143#discussion_r486723554

But this is `create unique index` so there can only be one validation per account. Did you mean `create index`?

Reply:
We chatted. I totally blanked that each validation will have a random `validation_id`... This works great.